resources/searchview/main.js

```diff
@@ -74,6 +74,10 @@
         }
         let topic = clickedLink.getAttribute("href") || "";
 
+        // Prevent opening links in the browser
+        event.preventDefault();
+        event.stopPropagation();
+
```

Comment:
Added `stopPropagation` to a couple of places as I was getting VS Code opening links in default browser.